backend/endpoints/rooms.py

```diff
@@ -61,3 +63,82 @@ async def create_room(
         status_code=status.HTTP_424_FAILED_DEPENDENCY,
         detail="unable to create room",
     )
+
+@router.delete("/org/{org_id}/members/{member_id}/rooms/{room_id}",
+    response_model=ResponseModel, )
+async def remove_member(org_id: str, member_id: str, room_id: str, mem_id: str):
+    """Removes a member from a room.
+
+    Fetches the room which the member is removed from from the database collection
+    Pops the member being removed from the room's members dict
+    Updates the database collection with the new room
+    *Returns the room dict if member was removed successfully
+    *while publishing to the user sidebar in the background
+
+    Args:
+        org_id (str): A unique identifier of an organisation
+        request: A pydantic schema that defines the room request parameters
+        member_id (str): A unique identifier of the member removing another member
+        room_id (str): A unique identifier of the room a member is being removed from
+        memb_id (str): A unique identifier of the member being removed from the room
+
+    Returns:
+        HTTP_200_OK (member removed from room): {room}
+    Raises
+        HTTP_404_NOT_FOUND: room or member not found
+        HTTP_424_FAILED_DEPENDENCY: member removal unsuccessful
+    """
+
+    room_obj = await get_room(org_id, room_id)
+
+    if not room_obj:
+        raise HTTPException(
+            status_code=status.HTTP_404_NOT_FOUND,
+            detail="room does not exist",
+        )
+    
+    if member_id not in room_obj["room_members"]:
+        raise HTTPException(
+            status_code=status.HTTP_404_NOT_FOUND,
+            detail="user not a member of the room",
+        )
+    
+    if member_id == mem_id:
```

Comment:
Where's member_id supposed to come from?
You haven't defined the variable prior to this line of code